packages/trpc/server/routers/loggedInViewer/integrations.handler.ts

```diff
@@ -10,23 +13,100 @@ type IntegrationsOptions = {
   input: TIntegrationsInputSchema;
 };
 
+type TeamQuery = Prisma.TeamGetPayload<{
+  select: {
+    id: true;
+    credentials: true;
+    name: true;
+    logo: true;
+  };
+}>;
+
+type TeamQueryWithParent = TeamQuery & {
+  parent?: TeamQuery | null;
+};
+
 export const integrationsHandler = async ({ ctx, input }: IntegrationsOptions) => {
   const { user } = ctx;
-  const { variant, exclude, onlyInstalled } = input;
-  const { credentials } = user;
+  const { variant, exclude, onlyInstalled, includeTeamInstalledApps, extendsFeature, teamId } = input;
+  let { credentials } = user;
+  let userAdminTeams: TeamQueryWithParent[] = [];
+
+  if (includeTeamInstalledApps || teamId) {
+    // Get app credentials that the user is an admin or owner to
+    userAdminTeams = await prisma.team.findMany({
```

Comment:
nit: Good candidate for getUserAdminTeams

Reply:
Good point. This query included the parent team so I add a `getParentInfo` to `getUserAdminTeams`